bower.json

```diff
@@ -18,7 +18,18 @@
     "shim"
   ],
   "license": "BSD-3-Clause",
-  "ignore": [],
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "src",
```

Comment:
Typically, I see the gulpfile passed down so you can modify and run your own for easy testing but isn't necessary. Package.json would be useful when everyone starts migrating to yarn